src/pages/[platform]/build-a-backend/data/aws-appsync-apollo-extensions/index.mdx

```diff
@@ -223,15 +192,143 @@ val authorizer = IamAuthorizer {
 //highlight-end
 ```
 
+</Block>
+
+<Block name="AppSync">
+
```

Comment:
Feels like we're missing a piece of text here. There should be something introducing the first code block.